q2cli/builtin/tools.py

```diff
@@ -195,7 +196,104 @@ def peek(path):
         click.echo(metadata.format)
 
 
-@tools.command('inspect-metadata',
+@tools.command(name='cast-metadata',
+               short_help='Designate metadata column types.',
+               help='Designate metadata column types.'
+                    ' Supported column types can be found on docs.qiime2.org.'
+                    ' Providing multiple file paths to this command will merge'
+                    ' the metadata.',
+               cls=ToolCommand)
+@click.option('--cast', required=True, metavar='COLUMN:TYPE', multiple=True,
+              help='Cast parameter for each metadata column that should'
+              ' contain a specified column type (supported types can be'
+              ' found on docs.qiime2.org). The required formatting for this'
+              ' parameter is --cast COLUMN:TYPE, repeated for each column'
+              ' and the associated column type it should be cast to in'
+              ' the resultant output.')
+@click.option('--ignore-extra', is_flag=True,
+              help='Cast parameters that do not correspond to any of the'
+              ' column names within the provided metadata will result in'
+              ' a raised error unless this flag is enabled.')
```

Comment:
🦑

The help text here is mostly describing what happens when the flag _isn't_ provided, which requires a little bit of mental overhead thinking about (thinking is no fun) - can you tweak the verbiage to describe what happens when the flag is enabled, primarily? This is likely an indicator that that root help text for the action is insufficient - moving some of this text up into the root help text should set you up well for succinct explanation here.